packages/ses/src/tame-symbol-constructor.js

```diff
@@ -0,0 +1,65 @@
+import {
+  Symbol,
+  entries,
+  fromEntries,
+  getOwnPropertyDescriptors,
+  defineProperties,
+  arrayMap,
+} from './commons.js';
+
+/**
+ * This taming provides a tamed alternative to the original `Symbol` constructor
+ * that starts off identical, except that all its properties are "temporarily"
+ * configurable. The original `Symbol` constructor remains unmodified on
+ * the start compartment's global. The tamed alternative is used as the shared
+ * `Symbol` constructor on constructed compartments.
+ *
+ * Starting these properties as configurable assumes two succeeding phases of
+ * processing: A whitelisting phase, that
+ * removes all properties not on the whitelist (which requires them to be
+ * configurable) and a global hardening step that freezes all primordials,
+ * returning these properties to their expected non-configurable status.
+ *
+ * The ses shim is constructed to eventually enable vetted shims to run between
```

Comment:
Implies that we’ll need to expose our intrinsics as well as the mechanism for altering the permits.

Reply:
I think so